src/features/dawnBreaker/components/Characters.tsx

```diff
@@ -0,0 +1,230 @@
+        </MapPlacement>
+      )}
+      {hoot && (
+        <MapPlacement x={hoot.x} y={hoot.y}>
+          <img
+            src={characterImages.hoot}
+            alt="Hoot"
+            style={{
+              width: `${PIXEL_SCALE * 15}px`,
+              transform: "translate(-3px, -10px)",
+            }}
+            // Remove brightness-50  if npc is in a bright part of the map
+            className={classNames("brightness-50", {
+              "cursor-pointer hover:img-highlight": hasDialogue("hoot"),
+            })}
+            onClick={
+              hasDialogue("hoot") ? () => handleDialogueOpen("hoot") : undefined
+            }
+          />
+        </MapPlacement>
+      )}
+      {maximus && (
+        <MapPlacement x={maximus.x} y={maximus.y}>
+          <img
+            src={characterImages.maximus}
+            alt="Maximus"
+            style={{ width: `${PIXEL_SCALE * 21}px` }}
+            // Remove brightness-50  if npc is in a bright part of the map
+            className={classNames("brightness-50", {
+              "cursor-pointer hover:img-highlight": hasDialogue("maximus"),
+            })}
+            onClick={
+              hasDialogue("maximus")
+                ? () => handleDialogueOpen("maximus")
+                : undefined
+            }
+          />
+        </MapPlacement>
+      )}
+      {snail && (
+        <MapPlacement x={snail.x} y={snail.y} width={1}>
+          <img
+            src={characterImages.snail}
+            style={{
+              width: `${PIXEL_SCALE * 15}px`,
+            }}
+            alt="Snail"
+            // Remove brightness-50  if npc is in a bright part of the map
+            className={classNames("brightness-50", {
+              "cursor-pointer hover:img-highlight": hasDialogue("maximus"),
+              "-scale-x-100": !!snail.flip,
+            })}
+            onClick={
+              hasDialogue("maximus")
+                ? () => handleDialogueOpen("maximus")
+                : undefined
+            }
+          />
+        </MapPlacement>
+      )}
+      {wendy && (
+        <MapPlacement x={wendy.x} y={wendy.y} width={1} height={2}>
+          <img
+            src={wendyGif}
+            style={{
+              width: `${PIXEL_SCALE * 15}px`,
+              bottom: `${PIXEL_SCALE * 1}px`,
+              left: `${PIXEL_SCALE * 0}px`,
+            }}
+            // Remove brightness-50  if npc is in a bright part of the map
+            className="brightness-50 absolute"
+            alt="Wood Nymph Wendy"
+          />
+        </MapPlacement>
+      )}
+      <Modal
+        show={showDialogue}
+        onHide={handleDialogueClose}
+        onExited={() => setSpeaking(null)}
+        onBackdropClick={() => setSpeaking(null)}
+        centered
+      >
+        {speaking && (
+          <CloseButtonPanel
+            onClose={handleDialogueClose}
+            bumpkinParts={speaking?.parts}
+          >
+            <div className="p-1 pt-0 flex">
```

Comment:
Can we also include a name at the top of the dialogue?

Bella says "Marcus I'm so scared", but it isn't quite clear who Marcus is